esp/public/media/scripts/program/modules/adminvitals.js

```diff
@@ -35,6 +35,18 @@ function fillStats(data)
 	}
     }
 
+    // Fill in the volunteer num data
```

Comment:
I'm less attached to it here as in #2368 since it's already a lot worse, but here, similarly, it would be nice while you're in the area to do a bit of cleanup and make it so we don't have to have 4 identical blocks of code for student/teacher/volunteer/class/hours data.  You could just define a function which takes an ID (e.g. `#stats_volunteers`) and a list of stats (e.g. `vitals.volunteernum`) and does all the work, and then just call it a bunch.